ui/app/src/components/Dashlet/Dashlet.tsx

```diff
@@ -21,24 +21,54 @@ import ExpandMoreIcon from '@material-ui/icons/ExpandMoreRounded';
 import useStyles from './styles';
 import AccordionDetails from '@material-ui/core/AccordionDetails';
 import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import { FormattedMessage } from 'react-intl';
+import Tooltip from '@material-ui/core/Tooltip';
+import RefreshRoundedIcon from '@material-ui/icons/RefreshRounded';
+import clsx from 'clsx';
 
 export type DashletProps = PropsWithChildren<
   Omit<AccordionProps, 'title'> & {
     title: React.ReactNode;
     headerRightSection?: React.ReactNode;
     icon?: ElementType;
+    onRefresh?(): void;
+    refreshDisabled?: boolean;
```

Comment:
**When** do you need to disable the refresh button? Should rather the whole interface be disabled?

Reply:
when there is an in progress action on the body, like an Fetch call, and the ui should be clickable to collapse or expand